pages/public_cloud/public_cloud_databases/databases_17_metrics_via_prometheus/guide.en-gb.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Public Cloud Databases - How to fetch service metrics with Prometheus
+excerpt: Find out how to setup a Prometheus exporter for your Public Cloud databases service
+updated: 2024-12-04
+---
+
+## Objective
+
+Public Cloud databases allow you to export service metrics using Prometheus.
+
+**This guide explains how to setup a Prometheus exporter for your Public Cloud databases service so that a Prometheus server can collect metrics.**
+
+## Requirements
+
+- A [Public Cloud databases service](https://www.ovhcloud.com/en-gb/public-cloud/databases/) up and running,
+- Access to the [OVHcloud API](https://eu.api.ovh.com/console/),
+- A [Prometheus](https://prometheus.io) server to collect metrics.
+
+## Instructions
+
+### Enable the Prometheus exporter for your service
+
+You can enable a Prometheus exporter for your service using the API endpoint:
+
+> [!api]
+>
+> @api {v1} /cloud PUT /cloud/project/{serviceName}/database/{engine}/{clusterId}
+>
+
+The boolean property `enablePrometheus` should be set to `true` to activate the Prometheus exporter for your service.
+
+### Retrieve the Prometheus exporter access info
+
+Use the following API endpoint to get the data needed to access the Prometheus exporter:
+
+> [!api]
+>
+> @api {v1} /cloud GET /cloud/project/{serviceName}/database/{engine}/{clusterId}/prometheus
+>
+
+You will also need to reset the credentials in order to obtain the password required for access:
+
+> [!api]
+>
+> @api {v1} /cloud POST /cloud/project/{serviceName}/database/{engine}/{clusterId}/prometheus/credentials/reset
+>
+
+### Configure your Prometheus server and start pulling metrics
+
+Use the data from the previous state to configure your Prometheus server and start pulling metrics for your service. Here is an example of such a Prometheus configuration for a MongoDB service:
+
+```yaml
+scrape_configs:
+- job_name: mongo-metrics
+  scrape_interval: 10s
+  metrics_path: /metrics
+  scheme : https
+  basic_auth:
+    username: prometheus
+    password: "xxx"
+  dns_sd_configs:
+    - names:
+      - '_prometheus._tcp.mongodb-7bfb5d40-oa85dff3e.database.cloud.ovh.net' 
+```
+
+The aiven-backed engines use a slightly different format:
```

Comment:
"Aiven-backed engines" is not comprehensible for most of our customer, most of them are not aware of a third party involved in the delivery of their service. We may prefer listing impacted engines.